src/FrameBase.re

```diff
@@ -57,3 +69,16 @@ external selectOneEval : (string, unit => unit) => Js.Promise.t('a) =
 [@bs.send.pipe : t]
 external selectAllEval : (string, unit => unit) => Js.Promise.t('a) =
   "$$eval";
+
+[@bs.send]
+external _addScriptTag : (t, tagOptions) => Js.Promise.t(ElementHandle.t) =
+  "addScriptTag";
```

Comment:
I'm wondering whether this should be an exposed function or not. If we expose `addScriptTag` only, then there would be no need for exposing `makeTagOptions` either. I'm inclined to call this external `addScriptTagWithOptions` or something like that and make it use `@bs.send.pipe` so there is a choice of using it for an API closer to upstream Puppeteer, or the shorter curryable option `addScriptTag`. If we choose not to expose this we will want to generate an interface file for this module.

Reply:
I've changed to us `@bs.send.pipe` in commit 
e0aec13